javascript/lib/api/src/client/handles/things.interfaces.ts

```diff
@@ -142,6 +172,17 @@ export interface HttpThingsHandle extends ThingsHandle {
    */
   putPolicyId(thingId: string, policyId: string, options?: MatchOptions): Promise<PutResponse<string>>;
```

Comment:
Any reason why policyId and definition are only used for http-requests and not for websocket-request?